docs/content/docs/recipes/signals.md

```diff
@@ -0,0 +1,328 @@
+  if (clientIdMap !== undefined) {
+      clientIdMap.delete(clientId);
+
+      //If UserID has no connected clients, remove user from local data
+      if (clientIdMap.size === 0) {
+          this.posMap.delete(member.userId);
+      }
+  }
+  this.emit("mousePositionChanged");
+});
+```
+`MouseTracker` must share information to all connected clients about where the local client's mouse position is. This is where `Signaler` comes in handy, as it specializes in communicating transient data to connected clients within a Fluid application.
+
+To track the client's mouse position, we use the `mousemove` event to obtain the client's x and y coordinates. To alert the connected clients of a mouse position change, we then submit a `mouseSignalType` signal with the client's userID and updated position value as the payload:
+
+```typescript
+window.addEventListener("mousemove", (e) => {
+    const position: IMousePosition = {
+        x: e.clientX,
+        y: e.clientY,
+    };
+    this.sendMouseSignal(position);
+});
+```
+```typescript
+/**
+* Alert all connected clients that there has been a change to a client's mouse position
+*/
+private sendMouseSignal(position: IMousePosition) {
+    this.signaler.submitSignal(
+        MouseTracker.mouseSignalType,
+        { userId: this.audience.getMyself()?.userId, pos: position },
+    );
+}
+```
+But what is the point of sending a signal if nobody is listening to it? To listen to the `mouseSignalType`, we use `Signaler`'s `onSignal` function. We then update the local data using the payload information from the signal. To display the new presence data, we then emit `mousePositionChanged` to let the view know to re-render:
+
+```typescript
+this.signaler.onSignal(MouseTracker.mouseSignalType, (clientId: string, local: boolean, payload: Jsonable) => {
+  this.onMouseSignalFn(clientId, payload);
+});
+```
+```typescript
+private readonly onMouseSignalFn = (clientId: string, payload: Jsonable) => {
+    const userId: string = payload.userId;
+    const position: IMousePosition = payload.pos;
+
+    let clientIdMap = this.posMap.get(userId);
+    if (clientIdMap === undefined) {
+        clientIdMap = new Map<string, IMousePosition>();
+        this.posMap.set(userId, clientIdMap);
+    }
+    clientIdMap.set(clientId, position);
+    this.emit("mousePositionChanged");
+};
+```
+
+## FocusTracker
+
+Let's now look at how the `FocusTracker` class uses the `Signaler` DataObject to share user focus infomation.
+
+The class defines the `focusSignalType` that will be sent and listened to the connected clients when there is a presence change:
+
+```typescript
+export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
+    private static readonly focusSignalType = "changedFocus";
+    /*...*/
+}
+```
+The class then initializes a local map of boolean values for all of the connected clients. The boolean denotes whether or not the client has focus or not:
```

Comment:
What does "the client has focus" mean? "Focus" is a UI term, but "client" isn't.